stable_baselines/gail/model.py

```diff
@@ -38,35 +38,41 @@ def __init__(self, policy, env, expert_dataset=None,
                  hidden_size_adversary=100, adversary_entcoeff=1e-3,
                  g_step=3, d_step=1, d_stepsize=3e-4, verbose=0,
                  _init_setup_model=True, **kwargs):
+        self.trpo = TRPO(policy, env, verbose=verbose, _init_setup_model=False, **kwargs)
```

Comment:
Why is this moved to before the super class constructor? (Seems like it'll work OK either way, and in general seems better to have things after the super call.)

Reply:
If I have it after the constructor call then `self.trpo` does not yet exist. This means that the magic methods will have to handle the specific case of `self.trpo` not yet existing, which seemed rather unnecessary considering an instance of GAIL must have an instance of TRPO.

Reply:
Then again, the attributes that are set in the constructor should belong to GAIL and GAIL alone, so I will go ahead and adjust this accordingly.